src/Tribe/Editor/REST/V1/Endpoints/Single_Ticket.php

```diff
@@ -254,6 +255,17 @@ public function add_ticket( WP_REST_Request $request, $nonce_action ) {
 			);
 		}
 
+		$is_paypal_ticket = $provider instanceof Tribe__Tickets__Commerce__PayPal__Main || $provider instanceof \TEC\Tickets\Commerce\Module;
+		$price            = (float)$body['price'];
```

Comment:
Avoid converting prices to Float (calculation errors), even if here is just to make a check it could cause problems in the future if someone uses this variable.

Do a couple checks, like `is_numeric` and something like using the float casting should be used directly on conditional so we dont create a new variable that could be misused.

Reply:
Creation of the variable was avoided. Instead I did the validation against `$body['price']` directly